WebInstall/explainer_cross_domain.md

```diff
@@ -193,6 +213,31 @@ A new web-manifest boolean key `allow_all_install_sources` signals that the appl
 This new manifest field will protect the app from being listed in undesirable repositories and give the developer absolute control about where do they want the PWA to be installed from. At best, the developer can allow the PWA to be installed from any site ("`*`"), at its most restrictive, it can only allow installing from the app's same scope. This field is only for the JS API and does not interfere with existing ways of installing PWAs through mechanisms like enterprise policies.
 
 If no `install_sources` are present in the manifest file, the default should be to not allow an app to be installed from cross-origin sites.
+=======
+    "all_all_install_sources": "true"
+}
+```
+>>>>>>> Stashed changes
+
+##### Fine tunning installation sources for an application
+
+In both cases of the default UA behaviour, developers can use the `install_sources` manifest field to have fine control over which specific origins can or can't install the application.
+
+
+```json
```

Comment:
Should there also be an example of how to combine `install_sources` and the `allow_all_install_sources`?